base/c.jl

```diff
@@ -643,8 +643,8 @@ function ccall_macro_lower(convention, func, rettype, types, args, nreq)
         sym = Symbol(string("arg", i, "root"))
         sym2 = Symbol(string("arg", i, ))
         earg, etype = esc(arg), esc(type)
-        push!(lowering, :($sym = Base.cconvert($etype, $earg)))
-        push!(lowering, :($sym2 = Base.unsafe_convert($etype, $sym)))
+        push!(lowering, :(local $sym = $(GlobalRef(Base, :cconvert))($etype, $earg)))
+        push!(lowering, :(local $sym2 = $(GlobalRef(Base, :unsafe_convert))($etype, $sym)))
```

Comment:
another nit pick, why can't we directly interpolate functions ? 
```suggestion
        push!(lowering, :(local $sym = $(Base.cconvert)($etype, $earg)))
        push!(lowering, :(local $sym2 = $(Base.unsafe_convert)($etype, $sym)))
```

Reply:
While we allow that, it's a bit of a misfeature. It's better for code to describe where an object comes from, or how it is computed, than to contain the object itself. One way to think of it is that code is a textual description, so it should only contain text.